.gitignore

```diff
@@ -3,3 +3,5 @@
 .Rdata
 .httr-oauth
 .DS_Store
+inst/doc
```

Comment:
Not sure did we miss a dot at the front?

Reply:
This is generate automatically so I do not think it is an issue.